.github/workflows/bootstrap.yml

```diff
@@ -79,6 +79,7 @@ jobs:
   # This way we can use it exclusively in branch protection rules
   # and abstract away the concrete jobs of the workflow, including their names
   bootstrap-post-job:
+    # This conditional overrides job cancellation
```

Comment:
Would be good to hint why we want this override.

Reply:
I'll do that when I figure it out. 🙃 (There's at least one place where the comment asks why we're doing it there; based on GHA docs, it seems inappropriate.)

Reply:
BTW, best I can figure is that it's used in jobs that are marked "required" in branch protection rules _and_ are part of the documentation-skip hack. Except that

1. it seems redundant, since cancellation of the job should already fail the required post-job;
2. there's those few places that aren't part of branch protection or the doc-skip hack and don't seem to gain anything at all from running on cancellation.